cli/commands/genesis/payload.go

```diff
@@ -85,6 +85,10 @@ func AddExecutionPayload(chainSpec ChainSpec, elGenesisPath string, config *cmtc
 		return errors.Wrap(err, "failed to read genesis doc from file")
 	}
 
+	//#nosec:G115 // genesis time won't ever be negative i.e. before Unix epoch (1970-01-01)
+	// Set the genesis time from app Genesis in the execution payload which gets converted to ExecutionPayloadHeader.
+	payload.Timestamp = uint64(appGenesis.GenesisTime.Unix())
+
```

Comment:
This is not related to the node-api. This sets the timestamp in the execution payload header in the beacon genesis file.

Reply:
This change can be removed. Since the `payload.Timestamp` is already set and determined by the eth-genesis file. If we overwrite it here (and the eth-genesis timestamp is actually different from the cl-genesis timestamp), we will produce a different execution payloaad header and people will have issues. Let's leave this CLI as is.

Reply:
understood, removed the cli change.